che-theia-plugins.yaml

```diff
@@ -741,3 +741,15 @@ plugins:
     extensions:
       - >-
         https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-eslint/vscode-eslint-2.1.1-1e15d3.vsix
+  - repository:
```

Comment:
It seems like these two extensions go together, maybe we merge the entries? `vscode-markdown-mermaid` could be the main one, and the syntax highlighting extension could just be a vsix in the `extensions` list.

Reply:
That what I wanted to do the first. But then decided to split them into different plugins just because different publishers.
From the other side, having only syntax highlighting without markdown preview does not have any sense.
So, combining them.